### PR TITLE
tests: reorganize fixtures

### PR DESCRIPTION
**Pull request by dtrifiro**
Prerequisite for real world tests using docker-compose and pytest-docker https://github.com/opendatahub-io/caikit-nlp-client/issues/33

### CODE REVIEW

**Review by z103cb — COMMENTED** [review]
This is Ok. Although it seems to have the same issues as #30. I am not sure if we are handling connecting to a GRPC server that does not have the needed messages.

File: tests/fixtures/http [hunk 1]
```diff
@@ -0,0 +1,4 @@
+@pytest.fixture(scope="session")
```
Comment (z103cb) [inline tests/fixtures/http]: I think this needs to have a .py extention

Comment (dtrifiro) [reply tests/fixtures/http]: It was actually added by mistake. Removed in the last push

**Review by z103cb — APPROVED** [review]

**Review by z103cb — APPROVED** [review]
LGTM

**Pull request merged**